apps/console/src/public/deployment.config.json

```diff
@@ -17,6 +17,7 @@
         }
     },
     "appBaseName": "console",
+    "centralDeploymentEnabled": false,
```

Comment:
Let's add this to framework too

Reply:
Do we need to add this to j2 file as well?